packages/www/package.json

```diff
@@ -12,7 +12,7 @@
     "docs-positions": "node ./generate-docs-positions.js",
     "docker:build": "docker build $(printf ' -t livepeerci/www:%s' ${DOCKER_TAGS:-master}) -f Dockerfile ../..",
     "docker:push": "for TAG in ${DOCKER_TAGS:-dev}; do docker push livepeerci/www:${TAG}; done",
-    "pkg": "pkg --out-path=bin --no-bytecode --public-packages \"*\" --public . --compress GZip",
+    "pkg": "pkg --out-path=bin --no-bytecode --public-packages \"*\" --public . --compress GZip ",
```

Comment:
Can revert the 2nd commit i guess

Reply:
after i get a clean master build i guess

Reply:
o tru